yucca/task_conversion/utils.py

```diff
@@ -20,12 +24,28 @@ def combine_imagesTr_from_tasks(tasks: Union[list, tuple], target_dir):
 
 def get_identifiers_from_splitted_files(folder: str, ext, tasks: list):
     if len(tasks) > 0:
-        uniques = np.unique([i[: -len("_000." + ext)] for task in tasks for i in subfiles(join(folder, task), join=False)])
+        uniques = np.unique(
+            [i[: -len("_000." + ext)] for task in tasks for i in subfiles(join(folder, task, suffix=ext), join=False)]
+        )
     else:
-        uniques = np.unique([i[: -len("_000." + ext)] for i in subfiles(folder, join=False)])
+        uniques = np.unique([i[: -len("_000." + ext)] for i in subfiles(folder, suffix=ext, join=False)])
     return uniques
 
 
+def dirs_in_dir(dir: str):
+    p = Path(dir)
+    return [f.name for f in p.iterdir() if f.is_dir() and f.name[0] not in [".", "_"]]
+
+
+def files_in_dir(dir: str):
+    p = Path(dir)
+    return [f.name for f in p.iterdir() if f.is_file() and f.name[0] not in [".", "_"]]
+
+
+def should_use(vol: nib.Nifti1Image):
```

Comment:
I would maybe use a slightly more informative name here (along the lines of the "read_any_file" discussion, haha)